Frontend/CO.CDP.OrganisationApp/Pages/Support/OrganisationApproval.cshtml.cs

```diff
@@ -13,11 +14,11 @@ public class OrganisationApprovalModel(
     public OrganisationWebApiClient.Organisation? OrganisationDetails { get; set; }
 
     [BindProperty]
-    [Required(ErrorMessage = "Select an option")]
+    [Required(ErrorMessage = nameof(StaticTextResource.Support_OrganisationApproval_ValidationErrorMessage))]
     public bool Approval { get; set; }
 
     [BindProperty]
-    [RequiredIf(nameof(Approval), false, ErrorMessage = "Tell the user why you could not approve")]
+    [RequiredIf(nameof(Approval), false, ErrorMessage = nameof(StaticTextResource.Support_OrganisationApproval_ErrorMessage))]
```

Comment:
The syntax has changed for this very recently so this isn't going to work any more. Please now start using this syntax


```[RequiredIf(nameof(Approval), false, ErrorMessageResourceName= nameof(StaticTextResource.Support_OrganisationApproval_ErrorMessage), ErrorMessageResourceType = typeof(StaticTextResource))]```